universe/ShipPart.cpp

```diff
@@ -100,6 +100,41 @@ namespace {
 
         return IncreaseMeter(meter_type, std::move(increase_vr));
     }
+
+    // create effectsgroup that increases the value of the part meter
+    // of type \a meter_type for part name \a part_name by the fixed
+    // amount \a fixed_increase
+    std::shared_ptr<Effect::EffectsGroup>
+    IncreaseMeter(MeterType meter_type, const std::string& part_name,
+                  float fixed_increase, bool allow_stacking = true)
+    {
+        auto increase_vr = std::make_unique<ValueRef::Constant<double>>(fixed_increase);
+        return IncreaseMeter(meter_type, part_name, std::move(increase_vr), allow_stacking);
+    }
+
+    // create effectsgroup that increases the value of the part meter
+    // of type \a meter_type for part name \a part_name by the fixed
+    // amount \a base_increase and the value of the game
+    // rule of type double with the name \a scaling_factor_rule_name
+    std::shared_ptr<Effect::EffectsGroup>
+    IncreaseMeter(MeterType meter_type, const std::string& part_name,
```

Comment:
perhaps call it something different... like `IncreaseMeterRuleScaled` or so...?

Reply:
Yes, probably a good idea to differentiate the call signatures a bit more.